src/chrome/content/rules/Podbean.com.xml

```diff
@@ -1,19 +0,0 @@
-<ruleset name="Podbean.com (partial)">
```

Comment:
Suggest to use `<ruleset name="Podbean.com (partial)" platform="mixedcontent"> ` as per [guideline](https://www.eff.org/https-everywhere/rulesets). It will disable the ruleset for browsers that block mixed content.